wazo_sysconfd/plugins/status/http.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2022 The Wazo Authors  (see the AUTHORS file)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from fastapi import APIRouter, Depends
+from fastapi.responses import JSONResponse
+from xivo.status import StatusAggregator
+
+router = APIRouter()
+
+
+@router.get('/status', status_code=200)
+def get_status(status: StatusAggregator = Depends(StatusAggregator)):
```

Comment:
Calling the `StatusAggregator` as `status` is looking for trouble :)

Reply:
Indeed, I will fix that